packages/block-editor/src/components/list-view/block.js

```diff
@@ -340,6 +345,8 @@ function ListViewBlock( {
 							} }
 							disableOpenOnArrowDown
 							__experimentalSelectBlock={ updateSelection }
+							expandedState={ expandedState }
+							expand={ expand }
```

Comment:
Why don't you just consume the context in the `BlockSettingsMenu` as well?

I guess it might re-render more often but I don't _think_ it's an expensive component is it?

Reply:
Sometimes the BlockSettingsMenu is a LeafMoreMenu, which doesn't live inside the ListView component. I thought it was better to pass the props than to make the LeafMoreMenu a part of the ListView component, so that we can pass in other BlockSettingsMenu components in the future.

Reply:
It could be a hook (`const [ expand, isExpanded ] = useListViewExpand` (I don't know what to call it)) that uses the context.

I think this is ok though. I think it might be good to look at a hook as part of refactoring the context providers as mentioned in that other PR. Maybe I'll try to tackle that.

Reply:
If we're doing that we can just use the context hook. Updated.